common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -290,9 +294,19 @@ def test_deferred_verification_enrollment(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect enrollment activated event
+        self.assertEqual(
```

Comment:
Ok I'll put it into this PR. Couple things on my plate today, but I'll ping you guys back with an update.